adapters/taboola/taboola.go

```diff
@@ -171,3 +252,25 @@ func evaluateDomain(publisherDomain string, request *openrtb2.BidRequest) (resul
 	}
 	return ""
 }
+
+func overrideBidRequestImp(originBidRequest *openrtb2.BidRequest, imp []openrtb2.Imp) (bidRequest *openrtb2.BidRequest) {
+	bidRequestResult := *originBidRequest
+	bidRequestResult.Imp = imp
+	return &bidRequestResult
+}
+
+func overrideEventTrackers(nativePayload *nativeResponse.Response) {
+	// convert evventrackers to the deprecated imptrackers and jstracker because it's not supported in native 1.1v
```

Comment:
nitpicks:  typo here.  `evventrackers` should be `eventrackers`

Reply:
@onkarvhanumante  fixed, thanks!